pedro-monteiro/1.i/index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Sistemas de Informação</title>
+</head>
+<body>
+    <h3>Sistemas de Informação</h3>
+    <p>O curso de Sistemas de Informação possui matriz curricular atual, com professores atuantes no mercado
+        profissional e altamente conceituados e titulados.<br>
+        O curso capacita e proporciona oportunidades de carreira para os diplomados e que atendem plenamente 
+        aos anseios do mercado, sempre exigente.</p>
+    
+    <h3>Atuação Profissional</h3>
+    <p>O Profissional de Tecnologia da Informação tem a capacidade de compreender os problemas do mundo real, 
+        aplicação dos conhecimentos adquiridos na solução de problemas, capacidade de análise sistêmica dos 
+        problemas, capacidade de síntese e de realização de projetos e, finalmente, capacidade de avaliação 
+        e julgamento. As habilidades pessoais desse profissional são: pensamento sistêmico, capacidade de 
+        resolução de problemas, pensamento crítico, disciplina pessoal, persistência, curiosidade, capacidade 
+        de auto aprendizado e aberto a mudanças. Habilidades Interpessoais: trabalho colaborativo, capacidade 
+        de comunicação, capacidade para resolução conjunta de problemas. Conhecimentos Técnicos: abstração, 
+        representação e organização da informação, uso de ferramentas computacionais para aplicação dos 
+        conhecimentos.</p>
+    
+    <h3>Corpo Docente</h3>
+    <p>Aline Heloisa Silva Villela<br>André Luiz Martins de Oliveira<br>André Luiz Paiva Pompeu<br>Artur Luís Ribas Barbosa
+        <br>Daniel P. Faria<br>Dionísio Ailton Pereira<br>Eduardo A. Rodrigues<br>Edy Segura<br>Estevan Marcelo Lopes
+        <br>Geovany Rosa Pires<br>Greciely Cristina da Costa<br>Hellen Patricia Morais Fonseca<br>Jane Aparecida De Oliv Silva
+        <br>Joelma Pereira de Faria<br>José Luiz da Silva<br>Júlio César Pereira<br>Kleber da Silva Garcia
+        <br>Márcio Emílio Cruz Vono de Azevedo<br>Paula Chiaretti<br>Roberto Ribeiro Rocha<br>Valéria Santos Paduan Silva</p>
+    
```

Comment:
To display the names above it is more appropriate to use the tags below as described in our [handout](https://edysegura.gitbooks.io/introducao-ao-desenvolvimento-web/content/introducao_a_linguagem_html/lista_nao_ordenada.html).

```html
<ul>
  <li>Aline Heloisa Silva Villela</li>
  <li>André Luiz Martins de Oliveira</li>
  .
  .
  .
</ul>
```